packages/strapi-plugin-content-manager/admin/src/components/CustomTable/Row.js

```diff
@@ -115,16 +123,22 @@ function Row({ canCreate, canDelete, canUpdate, isBulkable, row, headers, goTo }
       )}
       {headers.map(({ key, name, fieldSchema: { type }, cellFormatter }) => {
         const isMedia = type === 'media';
+        const isRelation = type === 'relation';
```

Comment:
Can you create a component for the `Td` in order to simplify the code?

`const Td = (props) => { }` (we might need to find a better naming).

Reply:
Sure

Reply:
WDYT about `<RowCell ... />` ? @soupette